boxsdk/object/terms_of_service.py

```diff
@@ -0,0 +1,132 @@
+        Get the terms of service user status.
+
+        :param user:
+            This is the user to get the status of the terms of service for. This defaults to current
+            user.
+        :type user:
+            :class:`User` or None
+        :returns:
+            A :class:`TermsOfServiceUserStatus` object
+        :rtype:
+            :class:`TermsOfServiceUserStatus`
+        """
+        url = self._session.get_url('terms_of_service_user_statuses')
+        additional_params = {
+            'tos_id': self.object_id,
+        }
+        if user is not None:
+            additional_params['user_id'] = user.object_id
+        box_response = self._session.get(url, params=additional_params)
+        response_object = box_response.json()
+        response = response_object['entries'][0]
+        return self.translator.translate(response['type'])(
+            session=self._session,
+            object_id=response['id'],
+            response_object=response,
+        )
+
+    def accept(self, user=None):
+        """
+        Create a terms of service user status.
+
+        :param user:
+            The :class:`User` to assign the terms of service to.
+        :type user:
+            :class:`User` or None
+        :returns:
+            A newly created :class:`TermsOfServiceUserStatus` object
+        :rtype:
+            :class:`TermsOfServiceUserStatus`
+        """
+        return self.set_user_status(is_accepted=True, user=user)
+
+    def reject(self, user=None):
+        """
+        Create a terms of service user status.
+
+        :param user:
+            The :class:`User` to assign the terms of service to.
+        :type user:
+            :class:`User` or None
+        :returns:
+            A newly created :class:`TermsOfServiceUserStatus` object
+        :rtype:
+            :class:`TermsOfServiceUserStatus`
+        """
+        return self.set_user_status(is_accepted=False, user=user)
+
+    def set_user_status(self, is_accepted, user=None):
+        """
+        Create a terms of service user status.
+
+        :param is_accepted:
+            Indicates whether a use has accepted or rejected a terms of service.
+        :type is_accepted:
+            `bool`
+        :param user:
+            The :class:`User` to assign the terms of service to.
+        :type user:
+            :class:`User` or None
+        :returns:
+            A newly created :class:`TermsOfServiceUserStatus` object
+        :rtype:
+            :class:`TermsOfServiceUserStatus`
+        """
+        url = self._session.get_url('terms_of_service_user_statuses')
+        body = {
+            'tos': {
+                'type': self.object_type,
+                'id': self.object_id,
+            },
+            'is_accepted': is_accepted,
+        }
+        if user is not None:
+            user_json = {
+                'type': user.object_type,
+                'id': user.object_id,
+            }
+            body['user'] = user_json
```

Comment:
You don't really need a temporary variable here; you could just assign the `dict` directly to `body['user']`